### PR TITLE
[Frontend][TFLite] support for FILL and SPLIT_V operators

### PR DESCRIPTION
**Pull request by maheshambule**
@siju-samuel, @shoubhik, @kevinthesun Please help in review.

### CODE REVIEW

File: python/tvm/relay/frontend/tflite.py [hunk 1]
```diff
@@ -1488,6 +1511,42 @@ def convert_split(self, op):
...
+        """SPLIT_V implementation."""
+        try:
+            from tflite.Operator import Operator
+        except ImportError:
+            raise ImportError("The tflite package must be installed")
+
+        assert isinstance(op, Operator)
+        input_tensors = self.get_input_tensors(op)
+
+        assert len(input_tensors) == 3, "input tensors length should be == 3"
```
Comment (siju-samuel) [inline python/tvm/relay/frontend/tflite.py:1524]: rephrase to -> input tensors length should be 3

File: python/tvm/relay/frontend/tflite.py [hunk 2]
```diff
@@ -1488,6 +1511,42 @@ def convert_split(self, op):
...
+        size_splits = list(self.get_tensor_value(input_tensors[1]))
+        size_splits = tuple(np.cumsum(size_splits)[:-1])
+
+        out = _op.split(in_expr, size_splits, axis=int(split_axis))
+        # Relay does not like a TupleWrapper of 1 element, further this
+        # only shows up with tf1.13 if we use a split with num_splits==1.
+        # In tf 1.14 this doesn't appear as it is automatically a reshape
+        # operation.
+        if isinstance(out, _expr.TupleWrapper):
+            if out.size == 1:
```
Comment (siju-samuel) [inline python/tvm/relay/frontend/tflite.py:1545]: This 2 if's can be combined

File: tests/python/frontend/tflite/test_forward.py [hunk 3]
```diff
@@ -1105,6 +1112,37 @@ def test_forward_zeros_like():
...
+    with tf.Graph().as_default():
+        input1 = array_ops.placeholder(dtype=value_dtype, name="input1", shape=dims)
+        # Fill op gets converted to static tensor during conversion
+        out = tf.fill(dims,  value_data)
+        out1 = tf.add(out, input1)
+        input1_data = np.random.uniform(0, 5, size=dims).astype(value_dtype)
+        compare_tflite_with_tvm([input1_data], ["input1"], [input1], [out1])
+
+
+def test_forward_fill():
```
Comment (siju-samuel) [inline tests/python/frontend/tflite/test_forward.py:1138]: test_forward_fill is not invoked from __main__

File: python/tvm/relay/frontend/tflite.py (hunk 1 above)
Comment (maheshambule) [reply python/tvm/relay/frontend/tflite.py]: Done

File: python/tvm/relay/frontend/tflite.py (hunk 2 above)
Comment (maheshambule) [reply python/tvm/relay/frontend/tflite.py]: done

File: tests/python/frontend/tflite/test_forward.py (hunk 3 above)
Comment (maheshambule) [reply tests/python/frontend/tflite/test_forward.py:1138]: oh missed it! Thanks. Done.

**Comment by maheshambule** [comment]
@siju-samuel Thanks. Made the changes.

**Comment by maheshambule** [comment]
@siju-samuel, Thanks. 
@kevinthesun, Please help in review and merge.

File: python/tvm/relay/frontend/tflite.py [hunk 4]
```diff
@@ -1067,6 +1069,27 @@ def convert_zeros_like(self, op):
 
         return out
 
+    def convert_fill(self, op):
+        """Convert TFLite FILL"""
+        try:
+            from tflite.Operator import Operator
+        except ImportError:
+            raise ImportError("The tflite package must be installed")
+
```
Comment (u99127) [inline python/tvm/relay/frontend/tflite.py:1078]: Please drop this , it's no longer needed as the assert has now been commonised.

File: python/tvm/relay/frontend/tflite.py [hunk 5]
```diff
@@ -1488,6 +1511,41 @@ def convert_split(self, op):
...
         return out
 
+    def convert_split_v(self, op):
+        """SPLIT_V implementation."""
+        try:
+            from tflite.Operator import Operator
+        except ImportError:
+            raise ImportError("The tflite package must be installed")
+
+        assert isinstance(op, Operator)
```
Comment (u99127) [inline python/tvm/relay/frontend/tflite.py:1521]: Likewise.

File: python/tvm/relay/frontend/tflite.py (hunk 4 above)
Comment (maheshambule) [reply python/tvm/relay/frontend/tflite.py]: Merged master and removed.

**Comment by maheshambule** [comment]
@u99127 I have merged the master and removed the duplicate code.

**Comment by maheshambule** [comment]
@u99127, @siju-samuel, @FrozenGene,  Could you please help in merging this PR.

**Comment by FrozenGene** [comment]
Thanks @maheshambule @siju-samuel @u99127 merged now

**Pull request merged**